### PR TITLE
Start

### PR DESCRIPTION
**Pull request by dmbarreiro**

### CODE REVIEW

**Review by ABohynDOE — APPROVED** [review]

**Comment by ABohynDOE** [comment]
Good job

**Comment by ABohynDOE** [comment]
Here is an article that describes the regex we need to use for the DOI check: https://www.crossref.org/blog/dois-and-matching-regular-expressions/

**Comment by dmbarreiro** [comment]
> Here is an article that describes the regex we need to use for the DOI check: https://www.crossref.org/blog/dois-and-matching-regular-expressions/

@ABohynDOE I adapted the regex using the one proposed by the article (adding the others brings little value and do not fully solves the problem, according to the article you would need dozens of regex to match all doi and you may still get some mismatches). 
You can have a look again and approve + merge if everything works for you. Let me know if you have more comments.

File: utils/validation/scripts/validation.py
Comment (ABohynDOE) [inline utils/validation/scripts/validation.py:13]: So this validates all the files present in the experiment folder using `experiments.yaml` for the validation ?

**Review by ABohynDOE — CHANGES_REQUESTED** [review]
Since the format of the yaml files has changed, we must change the `validation.py` file.
Maybe we should first all agree on the format of the `.yml` files before implementing the changes ?

**Comment by dmbarreiro** [comment]
> Since the format of the yaml files has changed, we must change the `validation.py` file. Maybe we should first all agree on the format of the `.yml` files before implementing the changes ?

Yeah, let's agree on the experiment schema before merging this

**Review by ABohynDOE — APPROVED** [review]

**Comment by ABohynDOE** [comment]
We should adapt the regex of the name of the variables so that it takes symbols into account.
Ex. of string failing: 'Amonium Nitrate (%g)'

**Comment by ABohynDOE** [comment]
We should have one file called `keywords.txt` where all the possible keywords are listed

**Comment by ABohynDOE** [comment]
@dmbarreiro I think it's ready for merging

**Review by ABohynDOE — APPROVED** [review]

**Pull request merged**
